client/src/components/Collections/ListWizard.vue

```diff
@@ -0,0 +1,280 @@
+            return "Configure auto-pairing";
+        }),
+        isValid: () => true,
+        isSkippable: () => !requiresPairing.value,
+    },
+    "list-builder": {
+        label: "Builder",
+        instructions: instructions,
+        isValid: () => builderInputsValid.value,
+        isSkippable: () => whichBuilder.value !== "list",
+    },
+    "list-paired-builder": {
+        label: "Builder",
+        instructions: instructions,
+        isValid: () => builderInputsValid.value,
+        isSkippable: () => ["list", "rules"].indexOf(whichBuilder.value) !== -1,
+    },
+    rules: {
+        label: "Builder",
+        instructions: computed(() => {
+            return "Assemble, label, and sort your list(s).";
+        }),
+        isValid: () => ruleState.value,
+        isSkippable: () => whichBuilder.value !== "rules",
+    },
+});
+
+const collectionTypeForPairedOrUnpairedBuilder = computed(
+    () => whichBuilder.value as SupportedPairedOrPairedBuilderCollectionTypes
+);
+
+const buildButtonLabel = computed(() => {
+    return "Build";
+});
+
+const pairedListType = computed(() => {
+    return whichBuilder.value == "list:paired_or_unpaired" ? "list:paired_or_unpaired" : "list:paired";
+});
+
+async function submit() {
+    if (collectionCreator.value) {
+        attemptCreate(collectionCreator);
+    }
+}
+
+async function onCreate(payload: CreateNewCollectionPayload) {
+    try {
+        await createHistoryDatasetCollectionInstanceFull(payload);
+        collectionCreated.value = true;
+    } catch (e) {
+        creationError.value = errorMessageAsString(e);
+    }
+}
+
+type RuleCreationRequestT = {
+    name: string;
+    collectionType: string;
+    elementIdentifiers: CollectionElementIdentifiers;
+    hide_source_items: boolean;
+}[];
+
+async function ruleOnAttemptCreate(createRequest: RuleCreationRequestT) {
+    for (const request of createRequest) {
+        const payload = createPayload(
+            request.name,
+            request.collectionType,
+            request.elementIdentifiers,
+            request.hide_source_items
+        );
+        await onCreate(payload);
+    }
+}
+
+function setWhichBuilder(newWhichBuilder: WhichListBuilder) {
+    whichBuilder.value = newWhichBuilder;
+}
+
+function onFilters(forwardFilter: string, reverseFilter: string) {
+    currentForwardFilter.value = forwardFilter;
+    currentReverseFilter.value = reverseFilter;
+}
+
+function goToAutoPairing() {
+    wizard.goTo("auto-pairing");
+}
+
+const ruleState = ref(false);
+
+function onRuleState(newRuleState: boolean) {
+    ruleState.value = newRuleState;
+}
+</script>
+
+<template>
+    <div v-if="currentHistoryId">
+        <div v-if="creationError">
+            <BAlert variant="danger">
+                {{ creationError }}
+            </BAlert>
+        </div>
+        <div v-if="collectionCreated">
+            <BAlert variant="success"> Collection created and it has been added to your history. </BAlert>
```

Comment:
I seem to never see the success message, even though the wizard is still mounted and the collection has been successfully created. I just see a blank center panel

Reply:
This alert needed a show attribute - that fixed this problem. Thanks for catching that!